rkt/run.go

```diff
@@ -92,6 +94,11 @@ func runRun(args []string) (exit int) {
 		return 1
 	}
 
+	if len(flagPodManifest) > 0 && (len(flagVolumes) > 0 || len(flagPorts) > 0 || flagInheritEnv || !flagExplicitEnv.IsEmpty() || rktApps.Count() > 0 || flagLocal) {
```

Comment:
perhaps we could provider a helper script/mode that will fetch the images and insert the id into a pod manifest?  On "first boot" of a pod, an end user may only care about versions, not the actual id's.